From d0ed89ba5902a5f301f102f8073098525e3c11ad Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Thu, 23 Jul 2020 22:10:18 +0200
Subject: [PATCH 2/2] PMOS: Hide Logout action

It doesn't work on pmOS and we don't need it.
---
 src/ui/top-panel.ui | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ui/top-panel.ui b/src/ui/top-panel.ui
index a91e9c4..7c6791b 100644
--- a/src/ui/top-panel.ui
+++ b/src/ui/top-panel.ui
@@ -17,7 +17,7 @@
         </child>
 	<child>
           <object class="GtkModelButton">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
             <property name="action-name">panel.logout</property>
             <property name="text" translatable="yes">Logout</property>
           </object>
-- 
2.27.0

